components/Panel/VotePanel/Details.tsx

```diff
@@ -116,8 +118,16 @@ export function Details({
     <Wrapper>
       <SectionWrapper>
         <RequestInfoIcons>
-          <ChainIcon chainId={augmentedData?.originatingChainId} />
-          <OoTypeIcon ooType={augmentedData?.originatingOracleType} />
+          <ChainIcon
+            chainId={
+              augmentedData?.originatingChainId as SupportedChainIds | undefined
+            }
+          />
+          <OoTypeIcon
+            ooType={
+              augmentedData?.originatingOracleType as OracleTypeT | undefined
```

Comment:
these have to get casted because we are relying on a superstruct type which doesnt have the nuances for these special types. perhaps this can be improved in another pass